### PR TITLE
Release v13 for all providers

### PR DESCRIPTION
**Pull request by rossf7**
Towards https://github.com/giantswarm/giantswarm/issues/5571

### CODE REVIEW

File: pkg/v13/key/key.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func CommonChartSpecs() []ChartSpec {
 		},
 		{
 			AppName:           "nginx-ingress-controller",
-			ChannelName:       "0-4-stable",
+			ChannelName:       "0-5-stable",
```
Comment (rossf7) [inline pkg/v13/key/key.go]: The change to use the new release channel.

File: pkg/v14/key/key.go [hunk 2]
```diff
@@ -105,7 +105,7 @@ func CommonChartSpecs() []ChartSpec {
 		},
 		{
 			AppName:           "nginx-ingress-controller",
-			ChannelName:       "0-4-stable",
+			ChannelName:       "0-5-stable",
```
Comment (rossf7) [inline pkg/v14/key/key.go]: Also for new WIP version.

File: service/controller/aws/v13/version_bundle.go [hunk 3]
```diff
@@ -8,10 +8,25 @@ func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
 			{
-				Component:   "TODO",
-				Description: "TODO",
+				Component:   "nginx-ingress-controller",
+				Description: "Updated to 0.23.0. More info here https://github.com/kubernetes/ingress-nginx/blob/master/Changelog.md.",
```
Comment (rossf7) [inline service/controller/aws/v13/version_bundle.go]: Double checking. Do we want to include the changelog here?

File: service/controller/aws/v13/version_bundle.go [hunk 4]
```diff
@@ -8,10 +8,25 @@ func VersionBundle() versionbundle.Bundle {
...
 				Kind:        versionbundle.KindChanged,
 			},
+			{
+				Component:   "nginx-ingress-controller",
+				Description: "Removed CPU and memory limits to Ingress Controller pods as per upstream recommendation.",
```
Comment (rossf7) [inline service/controller/aws/v13/version_bundle.go]: This was the best I could come up with. Open to improvements.

File: service/controller/aws/v13/version_bundle.go [hunk 5]
```diff
@@ -8,10 +8,25 @@ func VersionBundle() versionbundle.Bundle {
...
+				Kind:        versionbundle.KindAdded,
+			},
+			{
+				Component:   "nginx-ingress-controller",
+				Description: "Run single Ingress Controller pod per worker node.",
```
Comment (rossf7) [inline service/controller/aws/v13/version_bundle.go]: @pipo02mix This OK for your change?

Comment (pipo02mix) [reply service/controller/aws/v13/version_bundle.go]: yes

**Pull request merged**